addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -422,125 +601,169 @@
             }),
         ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+    </record>
+    <!-- 0.896 cent per kilowatt (to set as quantity) can be deduced -->
+    <!-- https://skat.dk/data.aspx?oid=2246453 -->
+    <record id="tax720" model="account.tax.template">
+        <field name="chart_template_id" ref="dk_chart_template"/>
+        <field name="sequence">720</field>
+        <field name="name">Elafgift</field>
+        <field name="description"></field>
+        <field name="amount">0.896</field>
+        <field name="amount_type">fixed</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
             }),
             (0,0, {
-                'factor_percent': -25,
+                'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8730'),
+                'account_id': ref('a8775'),
+                'plus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_electrical_tax')],
             }),
             (0,0, {
-                'factor_percent': 25,
+                'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8740'),
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
             }),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8750'),
+                'account_id': ref('a8775'),
+                'minus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_electrical_tax')],
             }),
             (0,0, {
                 'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8769'),
             }),
         ]"/>
     </record>
-
-    <record id="tax520" model="account.tax.template">
+    <!--  TODO: not found on official website -->
+    <record id="tax730" model="account.tax.template">
         <field name="chart_template_id" ref="dk_chart_template"/>
-        <field name="sequence">520</field>
-        <field name="name">EU Ydelseskøb</field>
+        <field name="sequence">730</field>
+        <field name="name">Naturgas- og bygasafgift</field>
         <field name="description"></field>
-        <field name="price_include" eval="0"/>
-        <field name="amount">100</field>
-        <field name="amount_type">percent</field>
+        <field name="amount">4.4490</field>
+        <field name="amount_type">fixed</field>
         <field name="type_tax_use">purchase</field>
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
```

Comment:
Hello @kenneth-vkd , I wasn't able to find the `Naturgas- og bygasafgift` tax on the skat website.
Would you have some source about that tax implementation ?
What I am looking for especially is for the amount of this tax in 2021.

Reply:
It takes a bit of hunting to find them, but here are the rates: https://skat.dk/skat.aspx?oid=2061637

Reply:
Sorry to bother you again but to me, it looks like this is an excise tax, not a "VAT like" one.
Am I right ?

Reply:
Yes. that is correct

Reply:
I'm considering deleting the tax from the PR.

The tax report in Odoo basically mean "VAT report" and I'm sure excise tax can't deduced on a VAT report and thus, there is no point in setting such a tax on a purchase line.

Maybe, a sale tax for this would be relevant for an energy reseller but it is **very** specific and should be set up by the client or its partner.

Reply:
It is not specifically VAT, but the big issue for users is that when they go to the SKAT selfservice portal to fill in the tax statement, a line will be shown for them to put in the `Naturgas- og bygasafgift`. So if we fx. leave it in the PR, but in an inactive state, it should still show on the tax report, but will only ever be used when specifically enabled on a database by the customer or the partner.
If it is removed from the PR. it will not be available unless specifically configured.
Competing ERP systems in Denmark has the tax report built to represent the tax statement from from the SKAT selfservice portal, so although most cases has these non-VAT taxes listed, they will have value of 0